### ISSUE TITLE
train model on the webface42m dataset, the loss of model no decrease

### ISSUE THREAD

**Issue by tinhchuquang** [DanJun6737/TopoFR#2]
![Screenshot from 2024-11-19 14-38-33](https://github.com/user-attachments/assets/3a0c99fe-1103-4b77-8b8e-4017f8723c19)
Thanks you, sharing code train and test.
But, I training with WebFace42, but from epoch 5, loss can't decrease. Can you show me your Config when training with webface42. Thanks.

**Comment by DanJun6737** [comment]
Hi @tinhchuquang 

Due to patent protection, we are temporarily not disclosing the configuration files and code used for training on the WebFace42 dataset. If you wish to achieve good accuracy on WebFace42, we recommend using the code with Partial FC provided in [InsightFace](https://github.com/deepinsight/insightface/tree/master/recognition/arcface_torch).

**Comment by tinhchuquang** [comment]
@DanJun6737 Thanks for reply.

**Comment by developer0hye** [comment]
Hi! @DanJun6737 

Thanks for sharing your works!

Could you share webface42 pretrained weights?
